### ISSUE TITLE
`github/no-useless-passive` false-negative when `passive: false` is passed

### ISSUE THREAD

**Issue by boris-petrov** [github/eslint-plugin-github#356]
```js
window.addEventListener('click', click, { passive: false });
```

This should trigger a `github/no-useless-passive` warning but doesn't.
